modules/virt-updating-virtio-drivers-windows.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
+
+:_content-type: PROCEDURE
+[id="virt-updating-virtio-drivers-windows_{context}"]
+= Updating VirtIO drivers on a Windows guest
+
+Update the `virtio` drivers by using Windows Update.
+
+.Prerequisites
+
+* A Windows guest OS with the `virtio` drivers must be attached to the VM.
+
+.Procedure
+
+. In the Windows Guest OS, use the `File Explorer` to navigate to *Settings* -> *Windows Update* -> *Advanced Options* -> *Optional Updates*
```

Comment:
I just noticed a couple nits. I might be wrong but I think we are not allowed to abbreviate OS. Also, please check for missing periods at the end of the steps and spell out "VM" the first time it is used.